app/assets/javascripts/Components/peer_reviews_manager.jsx

```diff
@@ -0,0 +1,334 @@
+              revieweeToReviewers={this.state.revieweeToReviewers}
+              groupIdToName={this.state.groupIdToName}
+              onReviewerChangeInRevieweeTable={this.updatedSelectedReviewersInRevieweesTable}
+              selectedReviewerInRevieweeGroups={this.state.selectedReviewerInRevieweeGroups}
+              loading={this.state.loading}
+              showSections={this.props.showSections}
+            />
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+
+class RawReviewersTable extends React.Component {
+  static columns = [
+    {
+      show: false,
+      accessor: '_id',
+      id: '_id'
+    },
+    {
+      Header: 'Reviewer Groups',
+      accessor: 'name',
+      id: 'name'
+    },
+    {
+      Header: 'Num Reviews',
+      accessor: 'groups',
+      className: 'groups',
+      filterable: false
+    }
+  ];
+
+  clearCheckboxes = () => {
+    this.refs.table.clearCheckboxes();
+  };
+
+  changeSection = (event) => {
+    this.clearCheckboxes();
+    this.setState({ sectionName: event.target.value });
+    this.props.refresh();
+  };
+
+  render() {
+    const hashmap = this.props.reviewerToNumReviews;
+    const groups_data = this.props.groups.map(function(group) {
+      let numReviews = 0;
+      if (hashmap.hasOwnProperty(group.id)) {
+        numReviews = hashmap[group.id];
+      }
+      return {
+        _id: group.id,
+        name: group.name,
+        groups: numReviews,
+        section: group.section
+      };
+    }.bind(this));
+
+    return (
+      <CheckboxTable
+        ref={(r) => this.checkboxTable = r}
+
+        data={groups_data}
+        columns={RawReviewersTable.columns}
+        defaultSorted={[
+          {
+            id: 'name'
+          }
+        ]}
+        loading={this.props.loading}
+        filterable
+
+        {...this.props.getCheckboxProps()}
+      />
+    );
+  }
+}
+
+
+class RawRevieweesTable extends React.Component {
+  getColumns = () => {
+    return [
+      {
+        show: false,
+        accessor: '_id',
+        id: '_id'
+      },
+      {
+        Header: 'Reviewee Groups',
+        accessor: 'name',
+        id: 'name',
+        filterable: true
+      },
+      {
+        Header: 'Assigned Reviewers',
+        accessor: 'members',
+        id: 'members'
+      },
+      {
+        Header: 'Num Assigned Reviewers',
+        accessor: 'count',
+        id: 'count'
+      },
+    ];
+  };
+
+  reviewerInRevieweeChange = (event) => {
+    const { onReviewerChangeInRevieweeTable } = this.props;
+    const isChecked = event.currentTarget.checked;
+    const reviewerGroupId = parseInt(event.currentTarget.getAttribute('data-reviewer-group-id'), 10);
+    const revieweeGroupId = parseInt(event.currentTarget.getAttribute('data-reviewee-group-id'), 10);
+    onReviewerChangeInRevieweeTable(reviewerGroupId, revieweeGroupId, isChecked);
+  };
+
+  checkboxShouldBeChecked = (reviewee_group_id, reviewer_group_id) => {
+    let returnVal = false;
```

Comment:
Don't do it like this. Instead, use a `if () { return ... } else { return ... }`